src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.exception.InvalidValueExeption;
+import core.basesyntax.service.CsvFileReaderService;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {

```

Comment:
Let's name tests accordingly to the actual classes names.
```suggestion
class CsvFileReaderServiceImplTest {

```